engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -1726,7 +1728,7 @@ class RuntimeErrorsTest
         xId,
         Api.MethodCall(Api.MethodPointer(moduleName, moduleName, "foo")),
         Api.ExpressionUpdate.Payload.Panic(
-          "9 (Integer)",
```

Comment:
@4e6, are you OK with these changes?

Reply:
I think this particular format when the type is reported together with the value, originally was used to make REPL errors more clear. I'm ok with the change.